misc-tools/dj_make_chroot.in

```diff
@@ -133,7 +133,7 @@ if [ "$DISTRO" = 'Debian' ]; then
 INCLUDEDEBS="ca-certificates"
 
 # Packages to install after upgrade (space separated):
-INSTALLDEBS="gcc g++ make default-jdk-headless default-jre-headless pypy pypy3 python3 locales"
```

Comment:
Are we not using python3 in the syntax check? If not this sounds fine but maybe @meisterT thinks otherwise

Reply:
That was changed in https://github.com/DOMjudge/domjudge/commit/75b2dded8f3b12bf0bddbfa3d2778e436867b69a